bin/codemods/src/single-tree-rendering.js

```diff
@@ -78,6 +84,39 @@ export default function transformer( file, api ) {
 	}
 
 	/**
+	 * Removes imports maintaining any comments above them
+	 *
+	 * @param {object} collection Collection containing at least one node. Comments are preserved only from first node.
+	 */
+	function removeImport( collection ) {
```

Comment:
I've always wondered why a handy tool like this isn't part of upstream -- `react-codemod`, specifically. Or is it?

Reply:
Good question! I had a look at `react-codemod` now and didn't see anything like this, but it's a good idea to try more actively look for generic helpers like this.

I might eventually move generic helpers like this to a separate codemod-utils file.